addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-mount-service.js

```diff
@@ -667,16 +667,53 @@ class EnvironmentMountService extends Service {
     return policyDoc.Statement;
   }
 
+  async _getPolicyDoc(possiblePolicyNames, roleName, workspaceRoleArn, iamClient) {
```

Comment:
nit: Since you are already passing "iamClient", would suggest accepting "policyNamePrefix" instead of the "workspaceRoleArn".

Reply:
Made the changes per our discussion